models/pull.go

```diff
@@ -1396,6 +1416,11 @@ func (pr *PullRequest) IsWorkInProgress() bool {
 	return false
 }
 
+// IsFilesConflicted determine if the  Pull Request has files conflicted.
```

Comment:
I would have ___determines___ here instead

Reply:
Going with @lafriks's suggestion with the translation

```suggestion

// IsFilesConflicted determines if the Pull Request has changes conflicting with the target branch.

```